.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +16,7 @@
 ### Resolves
 
 <!-- TODO: What Github issue does this resolve? Please include a link. -->
-Fixes 
+Fixes
```

Comment:
The space is actually part of the template here, because you can just click and start typing the issue number.

Reply:
What's hilarious is that I can't fix this in my IDE.  It strips all trailing spaces.  In the end I had to add an 'X' after 'Fixes', then use a hex editor to change the 'X' to a space.  This file is radioactive.